### PR TITLE
Change in Hunting Query Description - MS SQL server Database Audit

### PR DESCRIPTION
**Pull request by v-shukore**
Required items, please complete
   
   Change(s):
   - Repackaged solution for hunting query description

   Reason for Change(s):
   - Change in Hunting Query Description

   Version Updated:
   - 3.0.1

   Testing Completed:
   - yes
[SQL server DB Audit.docx](https://github.com/Azure/Azure-Sentinel/files/14633368/SQL.server.DB.Audit.docx)

### CODE REVIEW

File: Solutions/Microsoft Windows SQL Server Database Audit/Hunting Queries/SQL-Failed SQL Logons.yaml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 id: d98256d5-0c9a-4ffc-8618-66a3404412f8
 name: Failed Logon Attempts on SQL Server
 description: |
+    This query detects failed logons on SQL Server using the SQLEvent KQL Parser function.
```
Comment (rahul0216) [inline Solutions/Microsoft Windows SQL Server Database Audit/Hunting Queries/SQL-Failed SQL Logons.yaml:4]: This is enough "This query detects failed logons on SQL Server using the SQLEvent KQL Parser function."

File: Solutions/Microsoft Windows SQL Server Database Audit/Hunting Queries/SQL-MultipleFailedLogon_FromSameIP.yaml [hunk 2]
```diff
@@ -1,6 +1,9 @@
 id: 72727649-6445-46a3-b249-997a009fad89
 name: Failed Logon on SQL Server from Same IPAddress in Short time Span
 description: |
+    This query detects multiple failed logon attempts from the same IP within a short span of time. It relies on the SQLEvent KQL Parser function.
```
Comment (rahul0216) [inline Solutions/Microsoft Windows SQL Server Database Audit/Hunting Queries/SQL-MultipleFailedLogon_FromSameIP.yaml:4]: This is enough "This query detects multiple failed logon attempts from the same IP within a short span of time. It relies on the SQLEvent KQL Parser function."

File: Solutions/Microsoft Windows SQL Server Database Audit/Hunting Queries/SQL-MultipleFailedLogon_InShortSpan.yaml [hunk 3]
```diff
@@ -1,6 +1,9 @@
 id: aef212b5-c770-42e1-9abf-bc513e4e749c
 name: Multiple Failed Logon on SQL Server in Short time Span
 description: |
+    This query looks multiple failed logon attempts from the same IP within a short span of time. It relies on the SQLEvent KQL Parser function.
```
Comment (rahul0216) [inline Solutions/Microsoft Windows SQL Server Database Audit/Hunting Queries/SQL-MultipleFailedLogon_InShortSpan.yaml:4]: This is enough "This query looks multiple failed logon attempts from the same IP within a short span of time. It relies on the SQLEvent KQL Parser function."

File: Solutions/Microsoft Windows SQL Server Database Audit/Hunting Queries/SQL-New_UserCreated.yaml [hunk 4]
```diff
@@ -1,6 +1,9 @@
 id: 2b96760d-5307-44f0-94bd-8cf0ec52b1fb
 name: New User created on SQL Server
 description: |
+    This query detects new user creation from SQL Server using the SQLEvent KQL Parser function.
```
Comment (rahul0216) [inline Solutions/Microsoft Windows SQL Server Database Audit/Hunting Queries/SQL-New_UserCreated.yaml:4]: "This query detects new user creation in SQL Server using the SQLEvent KQL Parser function."

File: Solutions/Microsoft Windows SQL Server Database Audit/Hunting Queries/SQL-UserAdded_to_SecurityAdmin.yaml [hunk 5]
```diff
@@ -1,6 +1,9 @@
 id: 363ea6d1-b30d-4a44-b56a-63c3c8a99621
 name: User added to SQL Server SecurityAdmin Group
 description: |
+    This hunting query identifies user added in the SecurityAdmin group of SQL Server
```
Comment (rahul0216) [inline Solutions/Microsoft Windows SQL Server Database Audit/Hunting Queries/SQL-UserAdded_to_SecurityAdmin.yaml:4]: "This hunting query identifies user added in the SecurityAdmin group of SQL Server. It relies on the SQLEvent KQL Parser function."

File: Solutions/Microsoft Windows SQL Server Database Audit/Hunting Queries/SQL-UserDeletedFromDatabase.yaml [hunk 6]
```diff
@@ -1,6 +1,9 @@
 id: 7b8fa5f5-4f5b-4698-a4cf-720bbb215bea
 name: SQL User deleted from Database
 description: |
+    This hunting query identifies deletion of user from SQL Database
```
Comment (rahul0216) [inline Solutions/Microsoft Windows SQL Server Database Audit/Hunting Queries/SQL-UserDeletedFromDatabase.yaml:4]: "This hunting query identifies deletion of user from SQL Database. It relies on the SQLEvent KQL Parser function."

File: Solutions/Microsoft Windows SQL Server Database Audit/Hunting Queries/SQL-UserRemovedFromSecurityAdmin.yaml [hunk 7]
```diff
@@ -1,6 +1,9 @@
 id: f35b879c-c836-4502-94f2-c76b7f06f02d
 name: User removed from SQL Server SecurityAdmin Group
 description: |
+     This hunting query identifies user removed from the SecurityAdmin group of SQL Server
```
Comment (rahul0216) [inline Solutions/Microsoft Windows SQL Server Database Audit/Hunting Queries/SQL-UserRemovedFromSecurityAdmin.yaml:4]: "This hunting query identifies user removed from the SecurityAdmin group of SQL Server. It relies on the SQLEvent KQL Parser function."

File: Solutions/Microsoft Windows SQL Server Database Audit/Hunting Queries/SQL-UserRemovedFromServerRole.yaml [hunk 8]
```diff
@@ -1,6 +1,9 @@
 id: 5dd79877-8066-4ce4-ae03-eedd8ebf04f8
 name: User removed from SQL Server Roles
 description: |
+     This hunting query identifies user removed from a SQL Server Role.
```
Comment (rahul0216) [inline Solutions/Microsoft Windows SQL Server Database Audit/Hunting Queries/SQL-UserRemovedFromServerRole.yaml:4]: "This hunting query identifies user removed from a SQL Server Role. It relies on the SQLEvent KQL Parser function."

File: Solutions/Microsoft Windows SQL Server Database Audit/Hunting Queries/SQL-UserRoleChanged.yaml [hunk 9]
```diff
@@ -1,6 +1,9 @@
 id: 80a420b3-6a97-4b8f-9d86-4b43ee522fb2
 name: User Role altered on SQL Server
 description: |
+    This hunting query identifies user role altered on SQL Server
```
Comment (rahul0216) [inline Solutions/Microsoft Windows SQL Server Database Audit/Hunting Queries/SQL-UserRoleChanged.yaml:4]: "This hunting query identifies user role altered on SQL Server. It relies on the SQLEvent KQL Parser function."

**Review by rahul0216 — COMMENTED** [review]
Please address the inline comments.

**Review by rahul0216 — DISMISSED** [review]
Description changes look good.

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**